lib/node/failure_detector.ml

```diff
@@ -62,11 +62,11 @@ let rec pick_random_neighbors neighbors number_of_neighbors =
     else
       elem :: pick_random_neighbors neighbors (number_of_neighbors - 1)
 
-(** Updates a peer in the node's peer list with
```

Comment:
Keeping this as a documentation comment means that when you hover over this function's name elsewhere in the code, you can see the comment. I think this is quite useful :smile:.